### PR TITLE
Add support for the new Plex Movie agent

### PR DESCRIPTION
**Pull request by mza921**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2020-11-02
```
Comment (burkasaurusrex) [inline CHANGELOG.md:7]: Feels like this might collide with the other PR?

File: Dockerfile [hunk 2]
```diff
@@ -19,4 +19,4 @@ RUN \
 			/var/tmp/* \
 			/var/lib/apt/lists/*
 WORKDIR /app
-ENTRYPOINT ["python3", "plex_auto_collections.py", "--update"]
\ No newline at end of file
+ENTRYPOINT ["python3", "plex_auto_collections.py"]
```
Comment (burkasaurusrex) [inline Dockerfile:22]: Should we just change to `CMD` instead of `ENTRYPOINT`?

**Comment by burkasaurusrex** [comment]
It looks like `BeautifulSoup` should probably be added to `requirements.txt`? Are there any other new dependencies?

**Comment by burkasaurusrex** [comment]
Similar to `PyYAML` vs `ruamel.yaml`, we should probably default to one of `BeautifulSoup` or `lxml`. I think `BeautifulSoup` is more forgiving / maybe less performant, which I think it probably the right fit here.

**Comment by meisnate12** [comment]
i also think this should wait till the other pull is finished which im hoping to have done today

**Comment by meisnate12** [comment]
actually this can be pushed it doesn't any sections im working on we could do both at once

File: CHANGELOG.md (hunk 1 above)
Comment (mza921) [reply CHANGELOG.md:7]: It does collide, but I'm not sure how to handle it (I'm not a developer).

Comment (meisnate12) [reply CHANGELOG.md:7]: just do your pull first ill rebase my changes around yours

File: Dockerfile (hunk 2 above)
Comment (mza921) [reply Dockerfile:22]: I think `ENTRYPOINT` is okay here, since it's not meant to be overridden. The main thing was to just remove `--update`.

**Comment by mza921** [comment]
Thanks for catching the modules. Added `bs4`. `sqlite3` is standard.

**Comment by mza921** [comment]
I'm going to save the `PyYAML`/`ruamel.yaml` and `lxml`/`bs4` consolidation for another release. Performance of `lxml` vs `bs4` will be interesting once I throw a larger library at it, but I used `bs4` initially because it looked easier.

**Pull request merged**